src/openrct2/actions/WallPlaceAction.hpp

```diff
@@ -23,7 +23,34 @@
 #include "../world/Surface.h"
 #include "GameAction.h"
 
-DEFINE_GAME_ACTION(WallPlaceAction, GAME_COMMAND_PLACE_WALL, GameActionResult)
+
+class WallPlaceActionResult final : public GameActionResult
+{
+public:
+    WallPlaceActionResult()
+        : GameActionResult(GA_ERROR::OK, STR_CANT_BUILD_PARK_ENTRANCE_HERE)
+    {
+    }
+
+    WallPlaceActionResult(GA_ERROR err)
+        : GameActionResult(err, STR_CANT_BUILD_PARK_ENTRANCE_HERE)
+    {
+    }
+
+    WallPlaceActionResult(GA_ERROR err, rct_string_id msg)
+        : GameActionResult(err, STR_CANT_BUILD_PARK_ENTRANCE_HERE, msg)
+    {
+    }
+
+    WallPlaceActionResult(GA_ERROR error, rct_string_id msg, uint8_t* args)
+        : GameActionResult(error, STR_CANT_BUILD_PARK_ENTRANCE_HERE, msg, args)
+    {
+    }
+
+    TileElement* tileElement;
```

Comment:
Uninitialized variable.

Reply:
How should I resolve this, should I call the default TileElement constructor?

Reply:
Default value of nullptr will be fine

Reply:
Gotcha, committing now.